package.json

```diff
@@ -93,9 +93,6 @@
     ],
     "transformIgnorePatterns": [
       "/node_modules/(?!(@edx/paragon)/).*/"
-    ],
```

Comment:
These globals in the package.json did not work, and courseContext was not used. I removed it to avoid confusion.

Plus setting up globals in src/setupTest.js seems better than polluting the package.json